Pylint 2.x doesn't like passing object into class

Pylint 2.x says this:
```
************* Module adafruit_cursorcontrol.cursorcontrol
adafruit_cursorcontrol/cursorcontrol.py:46:0: R0205: Class 'Cursor' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
```
The line referenced:
https://github.com/adafruit/Adafruit_CircuitPython_CursorControl/blob/787b7cf1221b10ca603fd1cdb37f419deebf045d/adafruit_cursorcontrol/cursorcontrol.py#L45

I'm not sure why object is passed in, but I'm interested to find out why it was done.

Referencing main issue: adafruit/Adafruit_CircuitPython_Bundle#232

--- a/adafruit_cursorcontrol/cursorcontrol.py
+++ b/adafruit_cursorcontrol/cursorcontrol.py
@@ -42,7 +42,7 @@ import displayio
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_Cursor.git"
 
-class Cursor(object):
+class Cursor():
     """Mouse cursor interaction for CircuitPython.
 
     :param ~displayio.Display display: CircuitPython display object.

